lib/timeline/component/LineGraph.js

```diff
@@ -667,11 +666,13 @@ LineGraph.prototype._updateGraph = function () {
 
       //  at changeCalled, abort this update cycle as the graph needs another update with new Width input from the Redraw container.
       //  Cleanup SVG elements on abort.
-      if (changeCalled == true) {
+
+      /// [MODE change] remove this "optimization" so graph doesn't disappear when zooming in/out in browser
```

Comment:
how about the penalty?

Reply:
unsure. But this was my fix from before; I am just moving it here.